docs/tutorial.md

```diff
@@ -4,10 +4,23 @@ title: Step-by-Step Tutorial
 description: This document provides a step-by-step tutorial for the reSolve framework. Throughout this tutorial, you will iteratively develop a ShoppingList application and learn fundamental concepts of the reSolve framework.
 ---
 
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
 This document provides a step-by-step tutorial for the reSolve framework.
```

Comment:
Please rewrite without using "provides".

Reply:
Changed to "This document is a step-bystep tutorial"

Reply:
Thank you.